proto/ibc/applications/transfer/v1/query.proto

```diff
@@ -25,6 +25,11 @@ service Query {
   rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
     option (google.api.http).get = "/ibc/apps/transfer/v1/params";
   }
+
+  // DenomHash queries a denomination hash information.
+  rpc DenomHash(QueryDenomHashRequest) returns (QueryDenomHashResponse) {
+    option (google.api.http).get = "/ibc/apps/transfer/v1/denom_traces/{trace}";
```

Comment:
I think we need to regenerate the `swagger.yaml` to add this new path. @colin-axner for sure knows how to do it.

And question: should we have a different path? Something like `/ibc/apps/transfer/v1/denom_hashes/{trace}`?

Reply:
It should be `/ibc/apps/transfer/v1/denom_hashes/{trace}`, denom_traces is already a path. `make proto-swagger-gen` should work?

Reply:
sure, make proto-swagger-gen did work